packages/editor-ui/src/views/NodeView.vue

```diff
@@ -558,10 +601,17 @@ export default mixins(
 				if (saved) this.$store.dispatch('settings/fetchPromptsData');
 			},
 			openNodeCreator (source: string) {
+				if(this.createNodeActive) return;
```

Comment:
nitpick spacing..

Reply:
What do you mean? GitHub converts it to 4 spaces, but it's 2 spaces like the rest of the methods
<img width="1360" alt="CleanShot 2022-09-30 at 10 46 17@2x" src="https://user-images.githubusercontent.com/12657221/193231146-e6e36d70-f47a-4de6-99ec-f0c2ef3d5578.png">

Reply:
the space after `if`